ch02.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[ch02_getting_started]]
+== Getting Started
+
+// TODO intro paragraph - no empty headings!
+
+==== Choosing a Lightning Network Wallet
+
+* full nodes (c-lightning, eclair, lnd) + remote controls
```

Comment:
does eclair have a remote control? I am not aware of one.

Reply:
Remote control as in an RPC interface? It _does_ have a JSON-RPC interface which can be used to drive a node by a local program. Technically c-lightning is the same as well since their primary interface is accessible only via a unix socket, though I think they intend for user-defined plugs-in to fill that gap.